src/server/index.js

```diff
@@ -10,7 +10,7 @@ async function setLaunchUrl(element, { config, storage, storageProxy }) {
   const options = { ignoreAttributes: false, attributeNamePrefix: '$_' };
   const { manifest: parsedManifest } = xmlParser.parse(manifest.toString(), options);
   const { $_href: resourcePath } = parsedManifest.resources.resource;
-  const launchKey = `${path}/${root}/${resourcePath}`;
+  const launchKey = `repository/${element.repositoryId}/${path}/${root}/${resourcePath}`;
```

Comment:
Should we treat `repository/${element.repositoryId}/${path}` part of the path like its internal to Tailor (in such a way that elements are not aware of each of its components)? 🤔